src/main/util/randomAnnulusPoint.js

```diff
@@ -0,0 +1,5 @@
+export function randomAnnulusPoint(x, y, rMin, rMax) {
```

Comment:
This function generates points that are uniformly distributed over an annulus with given center and radii. It might be useful for other functions that need to generate particles.